mypy/reachability.py

```diff
@@ -261,6 +266,16 @@ def is_sys_attr(expr: Expression, name: str) -> bool:
     return False
 
 
+def is_platform_attr(expr: Expression, name: str) -> bool:
+    # Platform lib calls methods, it differs from sys lib
+    # sys.platform is of str class and platform.system is of
+    # function class
+    if isinstance(expr, MemberExpr) and expr.name == name:
+        if isinstance(expr.expr, CallExpr) and expr.expr.name == 'system':
```

Comment:
Shouldn't this be `if isinstance(expr.expr, CallExpr) and expr.expr.name == 'platform'`?